bundle/Form/FieldTypeHandler/Selection.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+
+use Ibexa\Contracts\Core\FieldType\Value;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\FieldDefinition;
+use Ibexa\Core\FieldType\Selection as SelectionValue;
+use Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use function array_flip;
+
+final class Selection extends FieldTypeHandler
+{
+    public function convertFieldValueFromForm($value): SelectionValue\Value
```

Comment:
Sync with interface signature.

Reply:
Fixed in bba42738

Reply:
Parameter name is still not the same as in the interface. Also, import the Value object and fix aliasing `Selection` type as `SelectionValue`.

Reply:
Fixed in e0905a8c